.github/workflows/ci.yml

```diff
@@ -99,10 +99,10 @@ jobs:
         run: |
           if [[ ${{ matrix.java }} == 17 ]]; then
             echo PROFILE=JDK17 >> $GITHUB_OUTPUT
-            echo MANDREL_IMAGE=ubi-quarkus-mandrel-builder-image:23.0-java17 >> $GITHUB_OUTPUT
+            echo MANDREL_IMAGE=ubi-quarkus-mandrel-builder-image:jdk-21 >> $GITHUB_OUTPUT
```

Comment:
Can't this be outside of if-else statement? As now it the same version.

Reply:
It could, but I have feeling that we may need/want to check another mandrel version soon. That's why I kept this in if-else.

Reply:
Ok let's keep this as it is.